application/dash_application/views/single_lane_wgs.py

```diff
@@ -591,3 +601,10 @@ def all_projects_requested(click):
     )
     def all_kits_requested(click):
         return [x for x in ALL_KITS]
+
+    @dash_app.callback(
+        Output(ids['library-designs-list'], 'value'),
+        [Input(ids['all-library-designs'], 'n_clicks')]
+    )
+    def all_library_designs_requested(click):
```

Comment:
(I spent too much time failing to figure out why this didn't work. Glad you succeeded! :laughing: )